css/modules/objects/post/_post.scss

```diff
@@ -436,7 +436,6 @@ $quarantine-VERTICAL_RHTYHM_LOOSE: 2rem;
   padding: 0.2rem 0.6rem;
   background-color: $COLOR_LIGHTEST;
   color: $COLOR_SECONDARY_DARK;
-  font-size: 1.4rem;
```

Comment:
removed this line b/c it is not being respected after compile, the .post pre rule above overrides it so that is why I changed line 424.